cfme/v2v/migrations.py

```diff
@@ -254,6 +298,84 @@ def is_displayed(self):
         return self.title.text == 'Migration Plan Wizard'
 
 
+class MigrationPlanRequestDetailsView(View):
+    migration_request_details_list = MigrationPlanRequestDetailsList("plan-request-details-list")
+    sort_type = SelectorDropdown('id', 'sortTypeMenu')
+    paginator_view = View.include(MigrationPlanRequestDetailsPaginationPane)
```

Comment:
This should pass `use_parent=True` so that the locators are resolved properly.

Reply:
@mshriver made this change